src/sdk/github.js

```diff
@@ -41,7 +41,7 @@ const load = ({ appId, gatekeeper, redirect, scope }) => new Promise((resolve, r
 
     const _redirect = parseAsURL(redirect)
     const searchParams = 'rslCallback=github'
-    let githubScopes = [ 'user' ]
+    let githubScopes = [ 'read:user' ]
 
     if (Array.isArray(scope)) {
```

Comment:
Your change is an improvement over the old one (and would fix my use case as well) but the concat part is the main problem. If someone supplies the scope property it should just override the default, not join it with a (possibly unwanted) default.

In my opinion it should be changed to this: 
```
    if (Array.isArray(scope)) {
      githubScopes = scope
    } else if (typeof scope === 'string' && scope) {
      githubScopes = scope.split(',')
    }
```

Reply:
Totally agree. Actually, I think it should be both. Should that be part of this PR or a separate one?

Reply:
I like the idea of empowering user to ope for default or define scopes by themselves. I will merge the code. While, replacing default to read:user is reasonable, to avoid lesser disruption, I will keep it as user. Profile and Email are common needs when using authentication and we wanted this library to be batteries included in that aspect. Thank you both.